SparseGrids/tsgOneDimensionalWrapper.cpp

```diff
@@ -67,7 +67,7 @@ OneDimensionalWrapper::OneDimensionalWrapper(const OneDimensionalMeta *meta, int
                 exit(1); // disabled by USE_XSDK_DEFAULTS
                 #else
                 max_level = meta->getCustom()->getNumLevels();
-                num_levels = max_levels;
+                num_levels = max_level;
```

Comment:
This change does not belong here, especially not within the same commit :/

Reply:
The bug prevents building in xSDK mode, I had to fix it to test the module. I guess you're right, I will put this in a separate pull-request.